Source/WebCore/dom/ContainerNodeAlgorithms.h

```diff
@@ -83,4 +90,10 @@ inline void disconnectSubframesIfNeeded(ContainerNode& root, SubframeDisconnectP
     disconnectSubframes(root, policy);
 }
 
+inline void updateCanDelayNodeDeletion(ContainerNode::CanDelayNodeDeletion& currentCanDelayDeletion, ContainerNode::CanDelayNodeDeletion newStatus)
```

Comment:
This doesn't need to be in the header file, does it? We can declare & define it in the cpp file instead.